src/textual/command.py

```diff
@@ -906,10 +907,14 @@ def _select_command(self, event: OptionList.OptionSelected) -> None:
 
     @on(Input.Submitted)
     @on(Button.Pressed)
-    def _select_or_command(self) -> None:
+    def _select_or_command(
+        self, event: Input.Submitted | Button.Pressed | None = None
```

Comment:
If this is a message handlers, then I don't think it will ever be None.

Reply:
It doubles as a message handler and an internally-called method; the message itself, until now, was throwaway. It's called elsewhere in the code with no message.

Reply:
(to add: this is one of the things I love about `@on` -- it removes the need for thin wrappers for messages to fire code that normally doesn't care one jot about the message itself)

Reply:
Oh I see. I would probably favor factoring out the common code rather than doing it that way. Not a blocker though.

Reply:
Yeah, I think I'm on the `@on`-is-awesome-lean-into-it-hard side of the fence (at least for my own code), but if this method had started out with an `event` parameter, I probably would have gone that way.